schema/library.schema.json

```diff
@@ -0,0 +1,47 @@
+{
```

Comment:
Aren't we missing `description_assertions`?

Reply:
That's right. Just added that